web/src/features/hfiCalculator/FormatTableAsCSV.ts

```diff
@@ -180,11 +181,13 @@ export class FormatTableAsCSV {
       dailyMeanIntensityGroups
     )
     const meanIntensityGroup = calculateMeanIntensityGroupLevel(dailyMeanIntensityGroups)
-    const calcPrepLevel = calculatePrepLevel(meanIntensityGroup)
+    const isMeanPrepLevel = true
```

Comment:
Instead of passing booleans, I think it would be more readable to just have 2 separate methods, `calculatePrepLevel` and `calculateMeanPrepLevel`. Internally they can reuse the same logic but naming this way makes the intent clear.

Reply:
Some more explanation and background: https://martinfowler.com/bliki/FlagArgument.html